src/containers/home/index.tsx

```diff
@@ -104,13 +123,23 @@ const Home: React.FC<HomeContainerProps> = ({ events, announcements }) => {
   }, []);
 
   const LandingCard = styled(Card)`
-    position: relative;
-    top: 30%;
+    position: absolute;
+    top: 40%;
```

Comment:
Percentages are scary, but I'm assuming you tested this on very large screen sizes too right 👀